### DISCUSSION TITLE
계정관련 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by phphll** [fusiondevkr/pl900#53] [1 Q&A]
![image](https://user-images.githubusercontent.com/68364002/131340422-675c2b56-302d-4bd9-bac8-f93be2a1c8a6.png)
현제 사용하던 jeongyoung.heo_seoul2021@techsoup.kr 계정이 프로젝트 완료후 삭제가되어 사용을 할수 없게되었습니다. (기존 fusiondevkr 등록계정)
이후 삭제가되어 개발자계정과 제 개인계정으로 접속사용을 해야할것 같은데 등록 가능할까요?

개인계정 ID는 phphll@naver.com 입니다.

**Comment by justinyoo** [comment]
앗, 현재까지 파워 플랫폼 기초 과정은 끝났으니 크게 걱정 안하셔도 되구요, 여기에 따로 적어두셨으니 나중에 참고하도록 하겠습니다!
